web/src/features/fbaCalculator/RowManager.ts

```diff
@@ -50,84 +50,91 @@ export class RowManager {
     tableRows: FBATableRow[]
   ): FBATableRow[] => {
     const reverseOrder = order === 'asc' ? 'desc' : 'asc'
+    const orderByWithEmptyAtBottom = (field: string, otherOrder?: Order) => {
+      const partition = _.partition(tableRows, x => !!_.get(x, field, null))
+      return _.concat(
+        _.orderBy(partition[0], field, otherOrder ? otherOrder : order),
```

Comment:
You could hear my neurons stretching and snapping while I looked at this. Could you add some documentation on this? What is partition, why is concat happening? What what?

Reply:
My brain is still pretty broken by this - why is `!!` necessary?

Reply:
`!!` is a (unfortunate?) javascript idiom to coerce a value into a boolean based on it's truthiness: https://dev.to/sanchithasr/what-is-the-double-bang-operator-in-javascript-4i3h

Reply:
damn you Javascript